### ISSUE TITLE
evalNWBin doesn't load file on window32

### ISSUE THREAD

**Issue by Arti3DPlayer** [nwjs/nw.js#5974]
I'm using `nw.js` 0.22.3 and use nwjc tool to compile my js file:

`nw.js_package/nwjs-sdk-v0.22.3-osx-x64/nwjc app/all.min.js app/app.bin`

Then i load bin file with following line:

```
<script>
        require('nw.gui').Window.get().evalNWBin(null, 'app/app.bin');
    </script>
```

After that i build app for all platforms with [nw-builder](https://github.com/nwjs-community/nw-builder). All works fine except win32. All functions from bin file not loaded. I tried build sdk version for win32 and run in console:
```require('nw.gui').Window.get().evalNWBin(null, 'app/app.bin');```
I don't get error, but functions didn't load

**Comment by kkm** [comment]
yeah can you test for me? https://github.com/nwjs/nw.js/issues/6001 please. 21.6 works well but above 22.0 doesnt work anymore

**Comment by Christywl** [comment]
Similar with #5995, a win32 specific issue, right?

**Comment by Christywl** [comment]
I try this on win32 device with nwjs-sdk-v0.27.3. After built the app with nw-builder, the win32 app doesn't work. It crashes. 
[616438b0-ed1f-4fe4-996c-ffaee2138b5f.zip](https://github.com/nwjs/nw.js/files/1597050/616438b0-ed1f-4fe4-996c-ffaee2138b5f.zip)

`win64/linux64/osx64` work fine.

**Comment by maikonmatheus** [comment]
This is caused when you use nwjc from a platform (win32 or win64) of NWJS and, after the build, try to load the .bin in another platform. Just use the nwjc from the same platform and will works.
